server/routes/agent_routes.ts

```diff
@@ -21,7 +21,7 @@ export function registerAgentRoutes(router: IRouter, assistantService: Assistant
           }),
           schema.object({
             dataSourceId: schema.maybe(schema.string()),
-            agentName: schema.string(),
+            agentConfigName: schema.string(),
```

Comment:
do we have any plugins using this API? they may need to change the name as well

Reply:
Yes, will sync with the plugin owners.